### ISSUE TITLE
k-ary trees, common parameters

### ISSUE THREAD

**Issue by busarobi** [busarobi/XMLC#19]
There are four PLT classes in the current cleanup branch:

PLTFH.java
PLTFHKary.java
PLTFHR.java
PLTFHRKary.java

The PLTFH was the first implementation which is based on feature hashing. Then it had been extended to keep track the parameters of FOBOS locally in each node of the tree, i.e.
`
this.Tarray[label]++;
this.scalararray[label] 
`

This version of the PLTFH is indicated by R, such as PLTFHR. Btw I have no clue why it is indicated by R :)
In fact it does not make too much sense to use a global T and scalar, therefore I recommend the original version being discarded. 

Moreover the version with R  had been made amenable to handle k-ary trees. These files got Kary endings. This makes the binary implementation obsolete.

In summary, in my opinion we should keep only PLTFHRKary.java. Regarding PLT.

What do you think about this issue?

**Comment by kdembczynski** [comment]
R states for separate regularization for each node. 

PLTFHRKary is the right class to keep.
